agent_group.yaml

```diff
@@ -208,66 +208,67 @@ agents:
         description: List of metasploit modules with their respective options
         value:
           - module: 'exploit/multi/http/atlassian_confluence_rce_cve_2023_22515'
-          - module: 'exploit/linux/http/dlink_hnap_header_exec_noauth'
+          # - module: 'exploit/linux/http/dlink_hnap_header_exec_noauth'
```

Comment:
Why comment, just delete if not needed!

Reply:
We might use them later if we add interactsh and trigger the exploit instead of just the check

Reply:
you can add them later!